addons/l10n_es_pos/tests/test_frontend.py

```diff
@@ -37,3 +37,12 @@ def get_number_of_regular_invoices():
         num_of_regular_invoices = get_number_of_regular_invoices() - initial_number_of_regular_invoices
         self.assertEqual(num_of_simp_invoices, 3)
         self.assertEqual(num_of_regular_invoices, 1)
+
+    def test_pos_session(self):
```

Comment:
Can you change the name of the test with something more explicit

Reply:
@robinengels the code has been updated, Please look into it. Thanks for the suggestions.